privacyidea/lib/policy.py

```diff
@@ -1094,19 +1109,22 @@ def ui_get_rights(self, scope, realm, username, client=None):
             admin_user = username
             admin_realm = realm
             user_object = None
+            extended_condition_check = CONDITION_CHECK.DO_NOT_CHECK_AT_ALL
```

Comment:
Why do we not check the conditions at all in case of an admin scope but we check userinfo and http header conditions in user scope?
Can You write a short comment explaining this?

Reply:
done in 0ccb69db